bangazonapi/fixtures/recommendation.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Can you add a command to load this file in the seed_data.sh?

Reply:
Of course